data/assets/scene/solarsystem/missions/jwst/L2.asset

```diff
@@ -0,0 +1,128 @@
+local assetHelper = asset.require('util/asset_helper')
+local transforms = asset.require('scene/solarsystem/sun/transforms')
+local circle = asset.require('util/circle').circle
+local kernels = asset.require('scene/solarsystem/missions/jwst/Lagrange_kernels').kernels
+
+local L2 = {
+    Identifier = "L2",
+    Parent = transforms.SolarSystemBarycenter.Identifier,
+    Renderable = {
+        Type = "RenderablePlaneImageLocal",
+        RenderableType = "Opaque",
+        Billboard = true,
+        Size = 400E4,
+        Texture = circle .. "/circle.png",
+        BlendMode = "Additive"
+    },
+    Transform = {
+        Translation = {
+            Type = "SpiceTranslation",
+            Target = "392", -- L2
+            Observer = "SUN",
+            Kernels = kernels .. "/L2_de431.bsp"
+        }
+    },
+    Tag = { "L2" },
+    GUI = {
+        Name = "L2",
+        Path = "/Solar System/Missions/JWST/Lagrange points",
+    }
+}
+
+local L2Large = {
+    Identifier = "L2Large",
+    Parent = transforms.SolarSystemBarycenter.Identifier,
+    Renderable = {
+        Enabled = false,
+        Type = "RenderablePlaneImageLocal",
+        RenderableType = "Opaque",
+        Billboard = true,
+        Size = 700E5,
+        Texture = circle .. "/circle.png",
+        BlendMode = "Additive"
+    },
+    Transform = {
+        Translation = {
+            Type = "SpiceTranslation",
+            Target = "392", -- L2
+            Observer = "SUN",
+            Kernels = kernels .. "/L2_de431.bsp"
+        }
+    },
+    Tag = { "lagrange_points" },
+    GUI = {
+        Name = "Large L2",
+        Path = "/Solar System/Missions/JWST/Lagrange points",
+    }
+}
+
+local L2Label = {
+    Identifier = "L2Label",
+    Parent = L2.Identifier,
+    Renderable = {
+        Enabled = true,
+        Type = "RenderableLabels",
+        LabelText = "L2",
+        FontSize = 14.0,
```

Comment:
Not super important, but would be nice if this label could also have `FontSize = 20.0`. Trying to reduce the number of different fonts sizes we use to get around a problem of the font atlas being filled when we load in too many labels (as part of a hitfix for #1206 ). 
The label size is the exponent of a 2^x scaling of the label, so setting it to just a little higher, maybe `6.9` or so should give about the same size

That said, this one will only load 2 symbols of that size, which is why it's not super important

Reply:
The label size for `RendeableLabels` is the exponent of a 10^x scaling of the label...